src/vm/olevariant.cpp

```diff
@@ -1275,7 +1275,8 @@ void OleVariant::MarshalBoolArrayOleToCom(void *oleArray, BASEARRAYREF *pComArra
 
 void OleVariant::MarshalBoolArrayComToOle(BASEARRAYREF *pComArray, void *oleArray,
                                           MethodTable *pInterfaceMT, BOOL fBestFitMapping,
-                                          BOOL fThrowOnUnmappableChar, BOOL fOleArrayIsValid)
+                                          BOOL fThrowOnUnmappableChar,
+                                          BOOL fOleArrayIsValid,SIZE_T szConst)
```

Comment:
We should call it cElements. There are other ways to control the size than szConst